modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeCommerceOrderConstantsCheck.testjsp

```diff
@@ -0,0 +1,16 @@
+<%--
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<%
+CommerceOrderConstants.PAYMENT_STATUS_AUTHORIZED;
+
+CommerceOrderConstants.PAYMENT_STATUS_PAID;
+
+CommerceOrderConstants.PAYMENT_STATUS_PENDING;
+%>
```

Comment:
@lazhcsilva, I have changed my mind. Let's elaborate a little bit on this test. Please, change to:

```
<%
	String constant = CommerceOrderConstants.PAYMENT_STATUS_AUTHORIZED;

	constant = CommerceOrderConstants.PAYMENT_STATUS_PAID;

	constant = CommerceOrderConstants.PAYMENT_STATUS_PENDING;
%>

```